src/idea.js

```diff
@@ -7,16 +7,16 @@ class Idea {
     this.star = star || false;
   }
 
-  saveToStorage(ideaCard) {
-    var stringifyObject = JSON.stringify(ideaCard);
-    localStorage.setItem(ideaCard.id, stringifyObject);
+  saveToStorage(array) {
+    var stringifyObject = JSON.stringify(array);
+    localStorage.setItem('allIdeas', stringifyObject);
```

Comment:
Nice job with the clean up here. Makes it much easier in the JS to figure out having a set name of `allIdeas` instead of finding specific ID instances